src/main/webapp/WEB-INF/views/car/create.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create car</title>
+</head>
+<body>
+<h1>Create car</h1>
+<form method="post" action="${pageContext.request.contextPath}/car/create">
+    Model <input type="text" name="model"><br>
+    Manufacturer <input type="number" name="manufacturer_id"><br>
```

Comment:
add required at least to model as there is no logic to insert empty model in db.